PWGMM/Mult/Tasks/CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ o2physics_add_dpl_workflow(heavy-ion-mult
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
                     COMPONENT_NAME Analysis)
 
+
```

Comment:
Please avoid pushing irrelevant or trivial changes.

Reply:
I am sorry, I did not notice that.